src/main/webapp/pages/projects/project_analyses.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html
+    xmlns:th="http://www.thymeleaf.org"
+    xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+    data-layout-decorate="~{projects/_base}"
+>
+<head>
+    <title th:text="${project.label} + ' - ' + #{project.nav.analysis}"></title>
+    <link rel="stylesheet" th:href="@{/dist/css/project-analyses.bundle.css}" />
+    <script th:inline="javascript">
+      window.PAGE = {
```

Comment:
Get rid of this

Reply:
Yup was going to get rid of this. Just wanted it in a working state before I changed this

Reply:
Removed in e28ce1f99099d12afd4ef7c9b50dae8c7bc964fb